src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/integration/RestrictedPatientIntegrationTest.kt

```diff
@@ -115,6 +118,7 @@ class RestrictedPatientIntegrationTest : IntegrationTestBase() {
   fun `remove restricted patient`() {
     prisonApiMockServer.stubCreateExternalMovement()
     prisonerSearchApiMockServer.stubRefreshIndex("A12345")
+    prisonerSearchApiMockServer.stubSearchByPrisonNumber("A12345")
```

Comment:
Was relying on a stub from a different test's web client - now it's more explicit